geomesa-utils/src/main/resources/org/locationtech/geomesa/geomesa-site.xml.template

```diff
@@ -193,6 +193,17 @@
     </property>
 
     <property>
+        <name>geomesa.stats.generate</name>
+        <value></value>
```

Comment:
default for the system property is null - the default for a datastore is handled in the data store configuration parameter 'generateStats'. Figured it would be confusing to have 2 defaults.